scionlab/scion_config.py

```diff
@@ -208,9 +202,13 @@ def _write_as_conf(self, elem_dir):
 
     def _write_systemd_services_file(self):
         instance_names = [r.instance_name for r in self._routers()] + \
-                         [s.instance_name for s in self._services()] + \
-                         ["sd%s" % self.AS.isd_as_path_str()]
-        unit_names = _get_systemd_services(instance_names)
+                         [s.instance_name for s in self._services()]
+
+        unit_names = _convert_instances_to_systemd_services(instance_names)
+        unit_names += _convert_static_to_systemd_services(self._static_services())
```

Comment:
I like that you've already pulled out the scion daemon into a separate case. Looking at this, I find this still rather convoluted, though and I think we can further simplify this by avoiding this `instance_name` alltogether and just build the entire list here:
```python
        ia = self.AS.isd_as_path_str()
        unit_names = ["scion-border-router@%s-%i.service" % (ia, router.instance_id) 
                      for router in self._routers()]
        unit_names += ["%s@%s-%i.service" % (SERVICES_TO_SYSTEMD_NAMES[service.type], ia, service.instance_id)
                       for service in self._services()]
        unit_names += ["%s@%s.service" % SERVICES_TO_SYSTEMD_NAMES[service.type]
                       for service in self._static_services()]
        ...
```
This may look a bit nasty but I think this is a lot easier to understand and modify for the next person to add something here.